mod/05_standard/set/type/uri.rb

```diff
@@ -0,0 +1,11 @@
+format do
+  include Phrase::Format
+
+  view :core do |args|
+    build_link _render_raw(args), render_title(args) || card.name
+  end
+
+  view :uri do |args|
+    build_link _render_raw(args)
```

Comment:
I think I'm not understanding you.  You are talking about aliasing uri and url, and I'm saying we leave that alone as far as view names.  The raw view of the new type already returns the bare uri, so we don't need another one for that, just a way to make it a link with the link also as the text.

How about {{a uri card|title:_raw}} or something else.  Some coding in title to say 'use the card's raw content', which is what we want for this feature.